shared-components/tl-date-time-card/tl-date-time-card-css.js

```diff
@@ -0,0 +1,17 @@
+import { css } from 'lit';
+
+export const dateTimeStyles = css`
+    .iconWrapper {
+        max-width: 13.4em;
+        min-height: 2.2em;
+        background: var(--primary-green);
+        border: 5px solid #1D0747;
```

Comment:
Almost the same colour as defined by `--color-secondary`... Can you explain the discrepancy?

Reply:
I prefer to follow design even if there is slight change in color because it is designers job to define that and mine to follow.